portfolio/src/main/webapp/script.js

```diff
@@ -28,10 +28,10 @@ function addRandomGreeting() {
   greetingContainer.innerText = greeting;
 }
 let turn;
```

Comment:
nit: you can delete this line and do
let turn = 'X';

at the beginning of initTic().